.azure-pipelines/azure-pipelines-win.yml

```
@@ -16,6 +16,10 @@ jobs:
     CONDA_BLD_PATH: D:\\bld\\
 
   steps:
+    - script: |
+        choco install cuda --version=11.0.3 -fdv -y --debug
+      displayName: Install Chocolatey Package: cuda --version=11.0.3
```

Maybe we need quotes? Seems like Azure is confused about parsing this (guessing the `:` throws it).

```suggestion
      displayName: "Install Chocolatey Package: cuda --version=11.0.3"
```

---

OK, let's try that.  Maybe it needs to be changed in smithy

---

Yeah seems like it does

---

Thanks Anthony! 😄